velox/type/TimestampConversion.h

```diff
@@ -44,6 +44,26 @@ constexpr const int32_t kMaxYear{292278994};
 constexpr const int32_t kYearInterval{400};
 constexpr const int32_t kDaysPerYearInterval{146097};
 
+// Enum to dictate parsing modes for date strings.
```

Comment:
Use /// for public comments. When documenting enums, put comments next to enum values.

```
/// Parsing modes for date and time strings.
enum class ParseMode {
   // ... describe kStrict...
   kStrict,

   // ... describe kNonStrict...
   kNonStrict,

...
};
```

Reply:
Done.